### PR TITLE
Updated interface for user

### PR DESCRIPTION
**Pull request by bluemelodia**
screen shot 2015-04-24 at 2 18 39 pm
screen shot 2015-04-24 at 2 18 47 pm
screen shot 2015-04-24 at 2 19 03 pm

### CODE REVIEW

File: app/forms/AddToWhitelistForm.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ class AddToWhitelistForm(Form):
     fake_user_image = StringField('Image', [image_with_same_name])
     user_type = RadioField('User Type',
                            [Required(message="Please select a user type.")],
-                           choices=[('editor', "Editor"),
+                           choices=[("fake_user", "Guest Writer"),
```
Comment (danrschlosser) [inline app/forms/AddToWhitelistForm.py]: single quotes, not double quotes please

File: app/forms/AddToWhitelistForm.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ class AddToWhitelistForm(Form):
...
+                                    ('editor', "Editor"),
                                     ('publisher', "Publisher"),
-                                    ('admin', "Admin"),
-                                    ("fake_user", "Fake User")])
+                                    ('admin', "Admin")])
```
Comment (danrschlosser) [inline app/forms/AddToWhitelistForm.py]: actually, change all of these to single quotes please

File: app/forms/EditUserForm.py [hunk 3]
```diff
@@ -32,4 +32,4 @@ class EditUserForm(Form):
                            choices=[('editor', "Editor"),
                                     ('publisher', "Publisher"),
                                     ('admin', "Admin"),
-                                    ("fake_user", "Fake User")])
+                                    ("fake_user", "Guest Writer")])
```
Comment (danrschlosser) [inline app/forms/EditUserForm.py]: Same here with single quotes

File: app/static/js/admin/users/users.js [hunk 4]
```diff
@@ -11,6 +11,25 @@ $(function() {
         }
     });
 
+    /* =======================================================================
+     * Table Highlight and Click
+     * ==================================================================== */
+    $('td').hover(function() {
+        var t = parseInt($(this).index()) + 1;
```
Comment (danrschlosser) [inline app/static/js/admin/users/users.js]: can we use a better varaible name than `t`?  what does this represent?  (maybe `columnNo`?)

File: app/static/js/admin/users/users.js [hunk 5]
```diff
@@ -11,6 +11,25 @@ $(function() {
...
+        $('td:nth-child(' + t + ')').addClass('highlighted');
+    },
+    function() {
```
Comment (danrschlosser) [inline app/static/js/admin/users/users.js]: When does this function get called?

File: app/static/js/admin/users/users.js [hunk 6]
```diff
@@ -11,6 +11,25 @@ $(function() {
...
+        var t = parseInt($(this).index()) + 1;
+        $('td:nth-child(' + t + ')').removeClass('highlighted');
+    });
+
+    var all_td = $('td');
```
Comment (danrschlosser) [inline app/static/js/admin/users/users.js]: change this variable name to `$td`.  Explanation: https://github.com/airbnb/javascript#jquery

File: app/static/scss/admin/users/users.scss [hunk 7]
```diff
@@ -14,6 +14,73 @@
         margin-top: 1rem;
         display: block;
     }
+    @media screen and (min-width: 641px) {
```
Comment (danrschlosser) [inline app/static/scss/admin/users/users.scss]: don't use pixel values here, instead use our media queries (like `$tablet-min`)

File: app/static/scss/admin/users/users.scss [hunk 8]
```diff
@@ -14,6 +14,73 @@
...
+                border: 1px solid #e0e0e0;
+            }
+            .highlighted {
+                background-color: #f5f5f5;
+            }
+            .clicked {
+                background-color: #efefef;
+            }  
+            th, td {
+                padding: 5px;
```
Comment (danrschlosser) [inline app/static/scss/admin/users/users.scss]: use `rem` instead of `px` here.

File: app/static/scss/admin/users/users.scss [hunk 9]
```diff
@@ -14,6 +14,73 @@
         margin-top: 1rem;
         display: block;
     }
+    @media screen and (min-width: 641px) {
+        .display-feature-table {
+            table, td, th {
+                overflow: hidden;
+                border: 1px solid #e0e0e0;
+            }
+            .highlighted {
```
Comment (danrschlosser) [inline app/static/scss/admin/users/users.scss]: All the common styles between the two media queries should be brought into one common rule outside of a media queries.

File: app/templates/admin/users/users.html [hunk 10]
```diff
@@ -60,6 +51,59 @@
             </a>
         </div>
     </div>
+    <div class="display-feature-table">
```
Comment (danrschlosser) [inline app/templates/admin/users/users.html]: You should use jinja to generate this!

Here are the steps:

- Define a data structure holding the data:

```html
{% set features = [
  ('Author blog posts',          [True, True, True, True]),
  ('Create and edit blog posts', [False, True, True, True]),
  ...
] %}
```

- Use jinja to generate it:

```html
{% for description, bools in features %}
<tr>
  <th>...</th>
  {% for bool in bools %}
  <td>
    {% if bool %}
    ...
    {% else %}
    ...
    {% endif %}
  {% endfor %}
</tr>
{% endfor %}
```

File: app/templates/events/event.html [hunk 11]
```diff
@@ -55,10 +55,8 @@
             {% if event.facebook_url %}
             <a href="{{event.facebook_url}}">
                 <div class="button blue">
-                    <div class="icon-facebook">
-                        <i class="fa fa-facebook"></i>
-                        RSVP on Facebook
-                    </div>
+                    <i class="fa fa-facebook"></i>
+                    RSVP on Facebook
```
Comment (danrschlosser) [inline app/templates/events/event.html]: Why did you remove this `<div>`?

File: app/static/js/admin/users/users.js (hunk 5 above)
Comment (bluemelodia) [inline app/static/js/admin/users/users.js]: It's called when the user hovers over a column in the table.

Comment (danrschlosser) [inline app/static/js/admin/users/users.js]: Ohhh, one is for hover in, the other is for hover out?

File: app/static/scss/admin/users/users.scss (hunk 9 above)
Comment (bluemelodia) [inline app/static/scss/admin/users/users.scss]: screen shot 2015-04-24 at 4 28 45 pm

Comment (bluemelodia) [inline app/static/scss/admin/users/users.scss]: There's a brief width range in which the table looks like this; is this okay?

Comment (danrschlosser) [inline app/static/scss/admin/users/users.scss]: That's fine.

File: app/static/js/admin/users/users.js (hunk 5 above)
Comment (bluemelodia) [inline app/static/js/admin/users/users.js]: Yep!

File: app/static/scss/admin/users/users.scss (hunk 9 above)
Comment (danrschlosser) [inline app/static/scss/admin/users/users.scss]: increase the width of the modal as we discussed

File: app/static/js/admin/users/users.js (hunk 5 above)
Comment (danrschlosser) [inline app/static/js/admin/users/users.js]: :+1:

File: app/static/scss/admin/users/users.scss (hunk 9 above)
Comment (bluemelodia) [inline app/static/scss/admin/users/users.scss]: There's still a small range where the table looks like this, but it looks better now. 
Making the width any larger would cause the "x" sign to go off-screen.
screen shot 2015-04-24 at 5 05 37 pm

File: app/templates/admin/users/users.html (hunk 10 above)
Comment (bluemelodia) [inline app/templates/admin/users/users.html]: A minor issue: the radio buttons move up and down as you shrink the screen. This happened even with the old table layout. Should I fix it or leave it?
screen shot 2015-04-24 at 5 10 24 pm
screen shot 2015-04-24 at 5 10 16 pm

**Comment by bluemelodia** [comment]
Note that there is a minor issue with radio buttons moving up and down (regardless of table format)

**Comment by danrschlosser** [comment]
You can leave it.  I think it's an issue with the tables, it will get fixed later.

File: app/templates/events/event.html [hunk 12]
```diff
@@ -73,10 +73,8 @@
             {% endif %}
             <a href="https://adicu.com/feedback?event={{event.title}}">
                 <div class="button blue">
-                    <div class="icon-comment">
-                        <i class="fa fa-comment"></i>
-                        Feedback
-                    </div>
```
Comment (danrschlosser) [inline app/templates/events/event.html]: Same question here -- any particular reason for removing this `<div>`?

**Comment by danrschlosser** [comment]
:+1:  after that one question about `app/templates/events/event.html`

**Comment by bluemelodia** [comment]
Added the header back
